src/app/containers/SwapTradeForm/components/arbitrage.tsx

```diff
@@ -31,9 +32,9 @@ export function Arbitrage() {
     // Only show component if you can earn > 0.001 BTC or > 1 USDT
     if (
       (data.USDT.rateToBalance.earn > 0.001 &&
-        symbolByTokenAddress(data.USDT.rateToBalance.to) === 'BTC') ||
+        symbolByTokenAddress(data.USDT.rateToBalance.to) === Asset.RBTC) ||
```

Comment:
@BetsyBraddock this data goes from backend and must be `BTC`, `RBTC` would fail assertion here?

Reply:
Yes you are right - backend uses BTC token abbreviation instead of RBTC

Reply:
Thanks, this is updated now